boxoffice/templates/cash_receipt_new.html.jinja2

```diff
@@ -0,0 +1,557 @@
+{%- from "macros_email.html.jinja2" import cta_button -%}
+{%- block content -%}
+  <br/>
+  <hr/>
+  <tr align="right">
+    <td class="receipt-header" colspan="4">
+      <p>{%- trans %}Receipt no. {% endtrans -%}<span>{{ order.receipt_no }}</span></p>
+    </td>
+  </tr>
+  <tr align="left">
+    <td class="invoice-header" colspan="4">
+      <p>{%- trans %}Hasgeek Learning Private Limited {% endtrans -%}</p>
+    </td>
+  </tr>
+  <tr>
+    <td style="width: 100%;" align="left" valign="top" colspan="4">
+      <!--[if (gte mso 9)|(IE)]>
+      <table cellpadding="0" cellspacing="0" border="0" role="presentation" style="width: 100%;">
+      <tr>
+      <td valign="top" style="width: 49.5%;">
+      <![endif]-->
+        <div style="display:inline-block;vertical-align:top;width: auto; margin-right: 68px">
+          <table>
+            <tr align="left">
+              <td class="invoice-label">
+                <p>{%- trans %}Address {% endtrans -%}</p>
+                {%- if org.details.get('address') %}
+                  <p>{{ org.details.get('address')|safe }}<p>
+                {%- endif %}
+              </td>
+            </tr>
+            <tr>
+              <td class="invoice-label">
+                <p>{%- trans %}PAN{% endtrans -%}</p>
+                {%- if org.details.get('pan') %}
+                  <p>{{ org.details.get('pan') }}</p>
+                {%- endif %}
+              </td>
+            </tr>
+          </table>
+        </div>
+      <!--[if (gte mso 9)|(IE)]>
+      </td><td valign="top" style="width:49.5%;">
+      <![endif]-->
+        <div style="display:inline-block;vertical-align:top;width: auto">
+          <table>
+            <tr>
+              <td class="invoice-label">
+                <p>{%- trans %}CIN {% endtrans -%}</p>
+                {%- if org.details.get('cin') %}
+                  <p>{{ org.details.get('cin') }}</p>
+                {%- endif %}
+              </td>
+            </tr>
+            <tr>
+              <td class="invoice-label">
+                <p>{%- trans %}Email {% endtrans -%}</p>
+                <p>{{ org.contact_email }}</p>
+              </td>
+            </tr>
+          </table>
+        </div>
+      <!--[if (gte mso 9)|(IE)]>
+      </td>
+      </tr>
+      </table>
+      <![endif]-->
+    </td>
+  </tr>
+  <tr align="left">
+    <td class="invoice-header" colspan="4">
+      <p>{%- trans %}Buyer details{% endtrans -%}</p>
+    </td>
+  </tr>
+  <tr>
+    <td style="width: 100%;" align="left" valign="top" colspan="4">
+      <!--[if (gte mso 9)|(IE)]>
+      <table cellpadding="0" cellspacing="0" border="0" role="presentation" style="width: 100%;">
+      <tr>
+      <td valign="top" style="width: 300px;">
+      <![endif]-->
+        <div style="display:inline-block;vertical-align:top;width: 49.5%">
+          <table>
+            <tr align="left">
+              <td class="invoice-label">
+                <p>{%- trans %}Name {% endtrans -%}</p>
```

Comment:
This could be without the hyphens and extra space:

```jinja2
<p>{% trans %}Name{% endtrans %}</p>
```

Or laid out in multiple lines for readability, with the hyphens added as necessary:

```jinja2
<p>
  {%- trans -%}
    Name
  {%- endtrans -%}
</p>
```

```jinja2
<p>
  {%- trans %}Name{% endtrans -%}
</p>
```